sdk/Lusid.Sdk/Api/TransactionPortfoliosApi.cs

```diff
@@ -1,27 +1,30 @@
  *
- * The version of the OpenAPI document: 0.11.3040
+ * The version of the OpenAPI document: 0.11.3028-MARK
  * Contact: info@finbourne.com
  * Generated by: https://github.com/openapitools/openapi-generator.git
  */
 
+
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-using RestSharp.Portable;
+using System.Net;
+using System.Net.Mime;
 using Lusid.Sdk.Client;
 using Lusid.Sdk.Model;
 
 namespace Lusid.Sdk.Api
 {
+
     /// <summary>
     /// Represents a collection of functions to interact with the API endpoints
     /// </summary>
-    public interface ITransactionPortfoliosApi : IApiAccessor
+    public interface ITransactionPortfoliosApiSync : IApiAccessor
```

Comment:
The addition of `Sync` is going to be a breaking change for the SDK. Do we want to keep it or should we remove `Sync` suffix from the template?